Bibliotheque nationale de France.js

```diff
@@ -528,34 +528,29 @@ var BnfClass = function() {
 	*/
 	this.getSelectedItems = function(doc) {
 		var items = new Object();
-		
-		var baseUri = /^(https?:\/\/[^\/]+)/.exec(doc.location.href)[1];
-		var cellPath = '//td[@class="mn_partienoticesynthetique"]';
-		var spanPath = './/span';
+		var cellPath = '//div[@class="liste-notices"]/div[@class="notice-item"]';
 		var cells = doc.evaluate(cellPath, doc, null, XPathResult.ANY_TYPE, null);
-		var cell = undefined;
-		var regexLink = /\s*window.location='([^']+)'\s*/;
-		
-		//Cell loop
 		while(cell = cells.iterateNext()) {
-			//Get link
-			var link = cell.attributes.item("onclick").textContent;
-			var url = baseUri + regexLink.exec(link)[1];
-			//Get title
-			var title = "";
-			var span = undefined;
-			var spans = doc.evaluate(spanPath, cell, null, XPathResult.ANY_TYPE, null);
-			//Span loop
-			while(span = spans.iterateNext()) {
-				if(title.length > 0) {
-					title += " – ";
+			var link = doc.evaluate('./div[@class="notice-contenu"]/a', cell, null, XPathResult.ANY_TYPE, null).iterateNext();
+			
+			var title = doc.evaluate('./h2', link, null, XPathResult.ANY_TYPE, null).iterateNext(); 
+			if (title) {
+				title = ZU.trim(title.textContent);
+			} else 	{
+				// 2016-01-26 : Sometimes there is no H2, we then need to get everything, example when searching for : 
+				// "Se souvenir de Tonnay-Charente"
+				title = ZU.trim(link.textContent);
 			}
-				title += Zotero.Utilities.trim(span.textContent);
+			
+			var documentYear = doc.evaluate('./div[@class="notice-ordre"]', cell, null, XPathResult.ANY_TYPE, null).iterateNext();
+			if ( documentYear ) {
+				title += " / " + documentYear.textContent;
```

Comment:
No good reason no, had just in mind it was done like that usually, never mind, will remove it.